bucket/haxe-dev.json

```diff
@@ -30,7 +30,7 @@
     ],
     "checkver": {
         "url": "https://github.com/HaxeFoundation/haxe/releases",
-        "re": "([\\d.]+-preview[\\d.]+)"
+        "re": "(\\d\\.\\d\\.\\d-preview\\.\\d|\\d\\.\\d\\.\\d\\-rc\\.\\d)"
```

Comment:
Why don't go with simple regex?

`([\d\.]+)-(rc|preview)[\d\.]+`

Reply:
I just started doing regex and I am going with what I can find and I didn't know we could do this so simply. Will change it.

Reply:
I tried your command and it doesn't work

Reply:
![Proof](https://i.imgur.com/qk0uZEn.png)

Reply:
The one in the pictures isn't the same as the one in your comment. But I will change it to what is in your picture. 
![image](https://user-images.githubusercontent.com/29337013/52506049-464f6d80-2bbb-11e9-8db4-7ad30befe1dd.png)

Reply:
It's same. It's only escaped to confort json.